### ISSUE TITLE
How is the trained model saved?

### ISSUE THREAD

**Issue by minSunlqb** [pylablanche/gcForest#18]
Hello,
Thank you very much for writing this program！But I have some questions.

 One, I can't find the trained model. I don't know if there is a log that saves the training model. It can be used directly to test the sample, instead of the test set being entered into the model along with the training set.
Two, I saw that you are using the out-of-bag error to test the model. If I want to use cross-validation instead of out-of-bag error, which piece of code should be modified. and I also want to ask if cross-validation is better than the out-of-bag error for this model.
Three, Where should the variety of classifiers be added? It seems that you only wrote one classifier in your code.
Four, I am very confused about the training of the decision tree of Cascade Forest. I don't know how the probability distribution is used as a feature for the training of decision trees.

Please forgive me for my poor English and so many questions，they really confused me for a long time.
Can you explain me these questions ?  Thank you in advance!
